lms/templates/static_templates/privacy.html

```diff
@@ -4,58 +4,145 @@
 <%inherit file="../main.html" />
 
 <%block name="pagetitle">${_("Privacy Policy")}</%block>
-<div class="container">
-<main id="main" aria-label="Content" tabindex="-1">
-    <section class="container about">
-        <h1>Polityka prywatności</h1>
 
-<h3>Postanowienia ogólne</h3>
 
-<p class="text-justify"><span lang="pl-PL">Ochrona osób fizycznych w związku z przetwarzaniem ich danych osobowych jest jednym z praw podstawowych każdej osoby. Minister Nauki i Szkolnictwa Wyższego przywiązuje szczególną wagę do poszanowania prywatności osób, których dane przetwarzają, bez względu na to czy zostały one pozyskane bezpośrednio od osoby, której dane dotyczą, jak i w przypadku, gdy zostały one pozyskane z innych źródeł. Niniejszy dokument polityki prywatności (dalej: Polityka) dostarcza osobom, których dane przetwarzane są w ramach platformy edukacyjnej Navoica informacji na temat sposobów ich pozyskiwania, przetwarzania oraz zabezpieczania.</span></p>
+<div id="skip-main">
+    <div class="privacy">
+        <header id="page-header">
+            <div class="jumbotron">
+                <div class="container">
+                    <h1 class="display-2 text-darkblack text-24">${_("Polityka prywatności Platformy Edukacyjnej NAVOICA")}</h1>
```

Comment:
Dlaczego jest tutaj gettext, treści w języku polskim? Chyba że ma to być tymczasowo, aczkolwiek pojawiło się to juz w źródlach

Reply:
Nie ma jeszcze wersji ang, jest tymczasowo.